CONTRIBUTING.md

```diff
@@ -56,6 +56,35 @@ This project follows the
 [Oracle Java code conventions](http://www.oracle.com/technetwork/java/codeconvtoc-136057.html)
 and the Scala conventions detailed above, preferring the latter.
 
+### Test
+There are two types of tests in this project. Unit tests that are written in Scala, and integration tests that are written in python. 
+We encourage writing integration tests if you have to choose between the two as that helps us ensure the plugin is working across different platforms. 
+
+#### Unit tests
+Unit-tests are located in com.nvidia:rapids-4-spark-tests_2.12. In order to run the unit-tests follow these steps
+1. Build the parent project. That will build the entire project including tests
+2. Running tests
+    2a. Issue the mvn command to run the tests like 'mvn -pl com.nvidia:rapids-4-spark-tests_2.12 test'. This will run all the tests in the tests module. 
+    2b. To run individual tests append '-DwildcardSuites=<Fully-qualified-test-name>' to the above command e.g. To run test ParquetWriterSuite in package com.nvidia.spark.rapids, issue 'mvn test -DwildcardSuites="com.nvidia.spark.rapids.ParquetWriterSuite"'
```

Comment:
Use backticks when quoting code or things the user should type verbatim.  This applies in multiple places.